### DISCUSSION TITLE
posts/2020-02-08/lazy-and-stupid/

### DISCUSSION THREAD

**Discussion by giscus** [tobyqin/tobyqin.github.io#2] [Comments]
# posts/2020-02-08/lazy-and-stupid/

如果你否定天分的存在，只强调勤奋努力，那就是蠢。但如果你因此认为勤奋努力没有用，那就是又懒又蠢。

https://www.tobyqin.cn/posts/2020-02-08/lazy-and-stupid/

**Comment by tobyqin** [comment]
是的，又懒又蠢。
